client_interface.py

```diff
@@ -50,16 +52,22 @@ def send_to_server(self, state):
                 async for msg in self.ws:
                     # Cycle "for" is finished when client disconnect from server
                     message = msg.json(loads=json.loads)
-                    if "game_state" in message.keys():
-                        state_for_client = State.from_dict(message)
-                        print(state_for_client)
-                    if "robot_data" in message.keys():
+                    if "game_state" in message:
+                        self.game_state = State.from_dict(message)
+                        # print(self.game_state)
+                    elif "robot_data" in message:
                         robot = Robot.from_dict(message)
                         self.state.robot = robot
                         print(robot)
+                    elif "dealt_cards" in message:
+                        self.state.dealt_cards = self.game_state.cards_from_dict(message)
```

Comment:
tento řádek opravdu nechápu. 
`self.state.dealt_cards` vím kde co je. Do `self.game_state` mám přiřazeno State.from_dict, tedy mi to načte roboty a board.  Tak jak si na tom můžu volat ještě `cards_from_dict`? @encukou

Reply:
jakože chápala bych `self.state.dealt_cards = State.cards_from_dict(message)`
a to self.game_state by mělo být v initu přidané?